docs/docs/addons-two-way-binding-helpers.md

```diff
@@ -1,16 +1,27 @@
 ---
 id: two-way-binding-helpers
-title: Two-Way Binding Helpers
-permalink: docs-old/two-way-binding-helpers.html
-prev: animation.html
-next: test-utils.html
+title: Two-way Binding Helpers (Legacy)
+layout: docs
+category: Add-Ons
+permalink: docs/two-way-binding-helpers.html
+prev: pure-render-mixin.html
+next: update.html
 ---
 
-`ReactLink` is an easy way to express two-way binding with React.
-
 > Note:
 >
-> ReactLink is deprecated as of React v15. The recommendation is to explicitly set the value and change handler, instead of using ReactLink.
+> 'react-addons-linked-state-mixin' is a legacy add-on. The recommendation is to explicitly set the value and change handler, instead of using `LinkedStateMixin`.
```

Comment:
Please don't put package names in apostrophes. Please use addon name rather than package name in warnings.

>`LinkedStateMixin` is a deprecated add-on. The recommendation is to explicitly set the value and change handler, instead of using `LinkedStateMixin`.